pylib/cqlshlib/cqlshmain.py

```diff
@@ -85,83 +86,11 @@
     webbrowser._tryorder.remove('xdg-open')
     webbrowser._tryorder.append('xdg-open')
 
-warnings.filterwarnings("ignore", r".*blist.*")
-
-DEFAULT_HOST = '127.0.0.1'
-DEFAULT_PORT = 9042
-DEFAULT_SSL = False
-DEFAULT_CONNECT_TIMEOUT_SECONDS = 5
-DEFAULT_REQUEST_TIMEOUT_SECONDS = 10
-
-DEFAULT_FLOAT_PRECISION = 5
-DEFAULT_DOUBLE_PRECISION = 5
-DEFAULT_MAX_TRACE_WAIT = 10
-
 if readline is not None and readline.__doc__ is not None and 'libedit' in readline.__doc__:
     DEFAULT_COMPLETEKEY = '\t'
```

Comment:
@bschoening can't be this moved to `Shell` as well?

Reply:
@smiklosovic I was planning to do that as part of [CASSANDRA-18914](https://issues.apache.org/jira/browse/CASSANDRA-18914).  Moving the logic and the globals got too mess, so this focuses mostly on refactoring the global constants.